templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>We should clear all the output of uninformative cells (i.e. keep the plots, remove everything else)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>